kube-runtime/src/reflector/store.rs

```diff
@@ -203,6 +259,24 @@ where
     (r, w)
 }
 
+/// Create a (Reader, Writer) for a `Store<K>` for a typed resource `K`
+///
+/// The resulting `Writer` can be subscribed on in order to fan out events from
+/// a watcher. The `Writer` should be passed to a [`reflector`](crate::reflector()),
+/// and the [`Store`] is a read-only handle.
+#[must_use]
+#[allow(clippy::module_name_repetitions)]
+#[cfg(feature = "unstable-runtime-subscribe")]
+pub fn store_shared<K>(buf_size: usize) -> (Store<K>, Writer<K>)
```

Comment:
maybe also note the buffer size here  and the intent behind it in a short sentence (or alternatively doc link to reflecthandle) since this is one of the main public interfaces

Reply:
Done. I detailed it a bit more in the dispatcher ctor and tried to summarise it more for `store_shared`.